audio/out/ao_coreaudio_chmap.c

```diff
@@ -65,6 +66,119 @@ static const int speaker_map[][2] = {
     { 0,                                       -1                 },
 };
 
+static const AudioChannelLayoutTag std_layouts[] = {
+    (100U<<16) | 1, // kAudioChannelLayoutTag_Mono
```

Comment:
maybe i missed it, why not use the enums directly?

Reply:
I need to iterate over this array; see `ca_find_standard_layout`. I don't think it's possible to iterate over enums.